src/ReportPortal.Shared/Observers/DefaultReportEventsObserver.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ReportPortal.Client.Abstractions.Models;
+using ReportPortal.Shared.Converters;
+using ReportPortal.Shared.Extensibility;
+using ReportPortal.Shared.Extensibility.ReportEvents;
+using ReportPortal.Shared.Extensibility.ReportEvents.EventArgs;
+using ReportPortal.Shared.Reporter;
+using System.Collections.Generic;
+
+namespace ReportPortal.Shared.Observers
+{
+    /// <summary>
+    /// Represents default report events observer.
+    /// </summary>
+    public class DefaultReportEventsObserver : IReportEventsObserver
```

Comment:
I propose to call it somehow meaningfully, `RequestsNormalizer`?

Reply:
renamed.